tasks/build/css.js

```diff
@@ -139,10 +139,16 @@ function buildCSS(src, type, config, opts, callback) {
   const packCompressed       = () => pack(type, true);
   packCompressed.displayName = 'Packing compressed CSS';
 
-  gulp.parallel(
-    gulp.series(buildUncompressed, packUncompressed),
-    gulp.series(buildCompressed, packCompressed)
-  )(callback);
+  if (config.compressed === undefined || config.compressed === 'both') {
```

Comment:
`compressed`  won't be `null` ?
If it will, `null` should be handled in if.

Reply:
Just `!config.compressed` should fetch undefined,null and false, doesnt it?

Reply:
> `compressed` won't be `null` ?
> If it will, `null` should be handled in if.

It can only if the user set the config value to ``null``. By the way, I've replicated the same structure as the [`rtl`](https://github.com/fomantic/Fomantic-UI/blob/a49c82330af6715f4de6332d40ed80f2fb070dbc/tasks/build/css.js#L159) config option where ``null`` is not taken into account.

> Just `!config.compressed` should fetch undefined,null and false, doesnt it?

``false`` should only produce uncompressed files. ``null`` and ``undefined`` can be bound to the original behavior of building both kind of files.

Reply:
I think it is common practice that handle `null` and `undefined` in same manner  in terms of configuration.
Unless intended for some reasons, `rtl` should be fixed in separate PR, and this PR should not repeat the error (not taking `null` into account)  in `rtl`.